src/main.cpp

```diff
@@ -28,5 +29,7 @@ int main(int argc, char **argv)
     setenv("XDG_CACHE_HOME", appCachePath.toUtf8().constData(), 1);
 
     LunaQmlApplication application(argc, argv);
+
+    QtWebEngine::initialize();
```

Comment:
Without it browser won't launch (the above error in journalctl). Tofe can explain more, I'm sure.